app/(api)/api/elections/route.ts

```
@@ -23,22 +27,32 @@ export async function GET(request: Request) {
+          },
+          {
+            state: "asc",
+          },
+        ],
       });
 
       return NextResponse.json(allElections);
     }
 
-    let elections: string | Election[] = [];
+    let elections: any[] = [];
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)